android/app/src/main/java/org/mozilla/magnet/MyAppPackage.java

```diff
@@ -22,6 +23,7 @@
         modules.add(new MagnetScannerReact(reactContext));
         modules.add(new PromptBluetoothReact(reactContext));
         modules.add(new ApiMagnetReact(reactContext));
+        modules.add(new NotificationListenerReact(reactContext));
```

Comment:
So that we can track notification-dismiss in JS analytics. Once we have native analytics it would be best to be done there as if the app is closed we can't track.